x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/action/StopDataFrameAnalyticsAction.java

```diff
@@ -81,8 +85,9 @@ public Request(String id) {
         public Request(StreamInput in) throws IOException {
             super(in);
             id = in.readString();
-            expandedIds = new HashSet<>(Arrays.asList(in.readStringArray()));
             allowNoMatch = in.readBoolean();
+            force = in.readBoolean();
+            expandedIds = new HashSet<>(Arrays.asList(in.readStringArray()));
```

Comment:
You can use `Set.of` in master, though it can make backporting a pain :)

Reply:
That's why I steered away from it. I think we'll have the time to replace those as we move on :-)